src/equations/linearized_euler_2d.jl

```diff
@@ -0,0 +1,184 @@
+Here, ``p`` is the pressure and ``v_1``,``v_2`` are the velocities.
+"""
+struct LinearizedEulerEquations2D{RealT<:Real} <: AbstractLinearizedEulerEquations{2, 3}
+  rho::RealT
+  lambda::RealT
+end
+    
+function LinearizedEulerEquations2D(rho::Real, lambda::Real)
+  return LinearizedEulerEquations2D(rho, lambda)
+end
+    
+function LinearizedEulerEquations2D(; rho::Real, lambda::Real)
+  return LinearizedEulerEquations2D(rho, lambda)
+end
+    
+varnames(::typeof(cons2prim), ::LinearizedEulerEquations2D) = ("v1", "v2", "p")
+varnames(::typeof(cons2cons), ::LinearizedEulerEquations2D) = ("v1", "v2", "p")
+     
+      
+# Set initial conditions at physical location `x` for time `t`
+"""
+    initial_condition_constant(x, t, equations::LinearizedEulerEquations2D)
+    
+A constant initial condition.
+"""
+function initial_condition_constant(x, t, equations::LinearizedEulerEquations2D)
+  v1 = 0.0
+  v2 = 0.0
+  p = 0.0
+  return SVector(v1, v2, p)
+end
+    
+"""
+    initial_condition_convergence_test(x, t, equations::LinearizedEulerEquations2D)
+    
+A smooth initial condition used for convergence tests in combination with
+[`source_terms_convergence_test`](@ref).
+"""
+function initial_condition_convergence_test(x, t, equations::LinearizedEulerEquations2D)
+  v1 = 2 + sin(2* pi * (x[1] - t))
+  v2 = 4 + cos(2* pi * (x[2] - t))
+  p = 6 + sin(2 * pi * (x[1] - t)) + cos(2 * pi * (x[2] - t))
+      
+  return SVector(v1, v2, p)
+end
+    
+"""
+    source_terms_convergence_test(u, x, t, equations::LinearizedEulerEquations2D)
+    
+Source terms used for convergence tests in combination with
+[`initial_condition_convergence_test`](@ref).
+"""
+function source_terms_convergence_test(u, x, t, equations::LinearizedEulerEquations2D)
+  c = equations.lambda * 2 * pi
+  a = 1/equations.rho * 2 * pi
+  omega = 2 * pi
+    
+  si = sin(omega * (x[2] - t))
+  co = cos(omega * (x[1] - t))
+    
+  du1 = -omega * co + a * co
+  du2 = omega * si - a * si
+  du3 = -omega * co + omega * si + c * co - c * si
+    
+  return SVector(du1, du2, du3)
+end
+    
+    
+# Calculate 1D flux for a single point
+@inline function flux(u, orientation::Integer, equations::LinearizedEulerEquations2D)
+  v1, v2, p = u
+    
+  if orientation == 1
+    f1 = 1/equations.rho * p
+    f2 = 0.0
+    f3 = equations.lambda * v1
+  else
+    f1 = 0.0
+    f2 = 1/equations.rho * p
+    f3 = equations.lambda * v2
+  end
+      
+  return SVector(f1, f2, f3)
+end
+    
+    
+# Calculate maximum wave speed for local Lax-Friedrichs-type dissipation
+@inline function max_abs_speed_naive(u_ll, u_rr, orientation::Integer, equations::LinearizedEulerEquations2D)
+  λ_max = sqrt(equations.lambda/equations.rho)
+end
+    
+    
+# Calculate 1D flux for a single point in the normal direction
+# Note, this directional vector is not normalized
+@inline function flux(u, normal_direction::AbstractVector, equations::LinearizedEulerEquations2D)
```

Comment:
This function should only be included if it has been tested and verified manually, and if a test is included.